app/panel/components/BuildingBlocks/DonutGraph.jsx

```diff
@@ -159,37 +223,31 @@ class DonutGraph extends React.Component {
 			}
 		});
 
-		// Draw graph
-		const chart = select(this.node)
-			.append('svg')
-			.attr('class', 'donutSvg')
-			.attr('width', '100%')
-			.attr('height', '100%')
-			.attr('viewBox', `0 0 ${width} ${height}`)
-			.attr('preserveAspectRatio', 'xMinYMin')
-			.append('g')
-			.attr('transform', `translate(${width / 2}, ${height / 2})`);
-		const trackerArc = arc()
-			.innerRadius(radius - 13)
-			.outerRadius(radius);
-		const trackerPie = pie()
-			.startAngle(-Math.PI)
-			.endAngle(Math.PI)
-			.sort(null)
-			.value(d => d.value);
-		const g = chart.selectAll('.arc')
-			.data(trackerPie(graphData))
-			.enter().append('g')
-			.attr('class', 'arc');
+		// CONNECT NEW DATA
+		const arcs = this.chartCenter.selectAll('g')
+			.data(this.trackerPie(graphData), d => d.data.id);
+
+		// UPDATE
+		arcs.select('path')
+			.transition()
+			.duration(d => (d.value / totalTrackers) * animationDuration)
+			.attrTween('d', function (d) {
+				const currentD = this.getAttribute('d');
+				const i = interpolate(currentD, d);
+				return function (t) {
+					return trackerArc(i(t));
+				};
+			});
 
-		g.append('path')
+		// ENTER
+		arcs.enter().append('g')
+			.attr('class', 'arc')
+		.append('path')
 			.style('fill', (d, i) => {
 				if (renderGreyscale) {
-					const greyTone = graphData.length > 1 ? 100 / (graphData.length - 1) * i * 0.01 : 0;
-					return this.colors.greyscale(greyTone);
+					return this.colors.greyscale(this.getTone(categoryCount, i));
 				} else if (renderRedscale) {
-					const redTone = graphData.length > 1 ? 100 / (graphData.length - 1) * i * 0.01 : 0;
-					return this.colors.redscale(redTone);
+					return this.colors.redscale(this.getTone(categoryCount, i));
```

Comment:
The colors don't change when the panel is open and I pause or restrict Ghostery.  But they do if I close the panel and reopen it.